nemo/collections/nlp/models/nlp_model.py

```diff
@@ -310,6 +319,20 @@ def load_from_checkpoint(
         checkpoint = None
         try:
             cls._set_model_restore_state(is_being_restored=True)
+
+            # dist checkpoint is a dir
```

Comment:
we support save/restore from a folder for untarred nemo as well, is this check safe?

Reply:
nvm, this is .ckpt path